orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/servergroup/strategies/CustomStrategy.groovy

```diff
@@ -33,17 +33,11 @@ class CustomStrategy implements Strategy {
 
   @Override
   List<Stage> composeBeforeStages(Stage parent) {
-    return composeFlow(parent)
```

Comment:
when i did the original composeflow refactor (https://github.com/spinnaker/orca/pull/3133/files) i took a shortcut and didn't fully use the `composeBefore/AfterStages` on strategy, this completes that work.

the thing to tripple check is that there were no "side-effects" in the compose calls since it used to be called during stage planning, but now it's not